pkgs/top-level/all-packages.nix

```diff
@@ -26853,6 +26853,11 @@ in
     boost = boost166;
   };
 
+  pinball = callPackage ../games/pinball {
+    autoreconfHook = with buildPackages;
+      autoreconfHook.override { automake = automake115x; };
```

Comment:
```suggestion
    autoreconfHook = buildPackages.autoreconfHook.override { automake = automake115x; };
```

Reply:
That would be incorrect, because automake115x has to come from buildPackages.

Reply:
You, the with is confusing.